src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunnerMode.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.RegularExpressions
+{
+    /// <summary>Represents the mode of execution for a <see cref="RegexRunner"/>.</summary>
+    internal enum RegexRunnerMode
+    {
+        /// <summary>The runner need only determine whether the input has a match; no additional information is required.</summary>
+        /// <remarks>This mode is used by Regex.IsMatch.</remarks>
+        Existence,
```

Comment:
This name is ambiguous to me. MatchExistenceRequired ? Or maybe CheckMatchExistence, GetMatchBounds, GetCaptures ?